test/widgets/lightbox_test.dart

```diff
@@ -194,8 +195,26 @@ class FakeVideoPlayerPlatform extends Fake
   }
 }
 
+class FakeHttpClient extends http.BaseClient {
+  final http.Response Function(http.Request) onRequest;
+
+  FakeHttpClient({required this.onRequest});
+
+  @override
+  Future<http.StreamedResponse> send(http.BaseRequest request) async {
+    final http.Response response = onRequest(request as http.Request);
+    return http.StreamedResponse(
+      Stream.fromIterable([response.bodyBytes]),
+      response.statusCode,
+      headers: response.headers,
+      reasonPhrase: response.reasonPhrase,
+    );
+  }
+}
```

Comment:
Hmm, I thought `test/api/fake_api.dart`'s implementation of `FakeHttpClient` should be sufficient for this. If not, consider extending that class instead.